src/driver/KoaDriver.ts

```diff
@@ -271,7 +271,7 @@ export class KoaDriver extends BaseDriver implements Driver {
             // note that we can't use error instanceof HttpError properly anymore because of new typescript emit process
             if (error.httpCode) {
                 options.context.status = error.httpCode;
-                response.status(error.httpCode);
+                // response.status(error.httpCode);
```

Comment:
It's really BAD when you comment part of code just to pass the test 😞 
> response.status is not a function
So change to `response.status = error.httpCode`!